src/terrain/clients/notifications.clj

```diff
@@ -46,13 +46,14 @@
 
 (defn send-team-join-notification
   "Sends a notification indicating that a user has requested to join a team."
-  [user-name user-email team-name admin message]
+  [user user-name user-email team-name admin message]
```

Comment:
That `user` is a username and `user-name` is a user's full name is slightly confusing, but I don't know if there's a better way to do it. Perhaps something like `user-id` instead of `user`? But `user-name` is always going to be confusing whether it's a username (which we use as an ID) or a user's name, I suppose.

Reply:
Heh, yeah, I briefly thought about that too.